core/src/test/java/brooklyn/entity/rebind/persister/BrooklynMementoPersisterToMultiFileTest.java

```diff
@@ -58,7 +59,10 @@ public void tearDown() throws Exception {
         mementoDir = Os.deleteRecursively(mementoDir).asNullOrThrowing();
     }
 
-    // to have this picked up in the IDE
-//    @Test public void noop() {}
-
+    @Test(groups="Integration")
+    public void testLoadAndCheckpointRawMemento() throws Exception {
+        // test here is deliberately no-op; 
+        // checkpoint routines not supported for this (deprecated) persister
```

Comment:
I prefer to do `throw new org.testng.SkipException("checkpoint routines not supported for this (deprecated) persister")`, but no strong feelings.